packages/react-composites/src/composites/ChatComposite/file-sharing/FileUploadButton.tsx

```diff
@@ -47,6 +47,16 @@ export const FileUploadButton = (props: FileUploadButtonProps): JSX.Element => {
     }
   });
 
+  const iconButtonClassName = mergeStyles({
```

Comment:
Similar to my comment above, let's not declare this code multiple times and re-use it
Tip: If the width and height need to be dynamic, you can convert this to a function that accepts the new width while having a default width of 1 rem

Reply:
Fileuploadbutton is in react-composites and Filedownloadcard/Fileuploadcard are in react-components. Do we want to access styles from components in composites, from what I observed is that we keep styles separate for both. We can make the changes if we want to use styles from composites, just wanted to check first :)

Reply:
Oh cool. I missed it my bad. Let's leave it like this for now.